Algorithm.CSharp/CompositeAlphaModelFrameworkAlgorithm.cs

```diff
@@ -66,38 +66,38 @@ public override void Initialize()
         /// </summary>
         public Dictionary<string, string> ExpectedStatistics => new Dictionary<string, string>
         {
-            {"Total Trades", "468"},
```

Comment:
it looks like these were generated using minute resolutions for the rsi/ema because that was their data resolution. A recent fix to the ema model made this now properly use the resolution parameter which defaults to daily and is now causing no signals to be emitted from the ema model.